app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <!--Set application-wide security config using base-config tag.-->
+    <base-config cleartextTrafficPermitted="true"/>
+</network-security-config>
```

Comment:
Please put TODO or smth to not forget to fix before merging to dev

Reply:
Done